pkg/api/aim/query/query.go

```diff
@@ -581,6 +612,57 @@ func (pq *parsedQuery) parseName(node *ast.Name) (any, error) {
 					}
 				},
 			), nil
+		case "re":
+			return callable(
```

Comment:
this should instead return an `attributeGetter` that switches between the 2 supported operations and returns a `callable`

Reply:
ook. let try to what you've suggested.

Reply:
done